README.md

```diff
@@ -55,3 +55,6 @@ Effect:
 
 * Downloads sample PDFs
 * Converts PDF by running the _elifesciences/sciencebeam_ container
+
+## Front End
+ * To setup the Front End so you see styling changes then look at the README file in the demo folder (/demo/README.md).
```

Comment:
I hope you don't mind a rewording suggestion (to make it clear that it avoid having to rebuild the docker image). Also Markdown linting:

```suggestion

* Please read the [README file in the demo folder](demo/README.md) to setup the Front End for seeing styling changes, without needing to rebuild the Docker image.
```